src/core/session.ts

```diff
@@ -246,8 +249,8 @@ export class Session implements NavigatorDelegate {
     return isAction(action) ? action : "advance"
   }
 
-  linkIsVisitable(link: Element) {
-    const container = link.closest("[data-turbo]")
+  turboIsEnabled(element: Element) {
```

Comment:
Let's call this `elementIsNavigable`.

Reply:
If we change the signature to `element?: Element` we can avoid the `submitter` conditional expression in `willSubmitForm`.